pylint/testutils/output_line.py

```diff
@@ -58,10 +58,10 @@ def __init__(self, row, exception):
 class OutputLine(NamedTuple):
     symbol: str
     lineno: int
-    column: int
+    column: str
```

Comment:
```suggestion
    column: Union[str, int]
```

Reply:
Shouldn't we try and refactor this to be only one of the two? Seems weird to allow both, but perhaps I am missing something obvious here!

Reply:
Do you want me to commit this suggestion? Or keep as is?

Reply:
Well no you were right we're reading from the functional result file so it's always a string even if it should be castable to int.